samples/excel/45-named-item/create-and-use-named-item-for-range.yaml

```diff
@@ -0,0 +1,171 @@
+id: excel-create-and-use-named-item-for-range
+name: Create and use named range item
+description: Create and use named range item
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |-
+        $("#add-name").click(() => tryCatch(addName));
+        $("#format-named-range").click(() => tryCatch(formatRangeByName));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function addName() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const headerRange = sheet.getRange("A1:E1");
+                sheet.names.add("ExpensesHeader", headerRange);
+                const namedItems = sheet.names.load("name, type");
+
+                await context.sync();
+
+                let namedItemsList = `This workbook contains ${namedItems.items.length} named item(s):`;
+                for (let i = 0; i < namedItems.items.length; i++) {
+                    namedItemsList += JSON.stringify(namedItems.items[i]);
+                }
+                OfficeHelpers.UI.notify(namedItemsList);
+
+                await context.sync();
+            });
+        }
+
+        async function formatRangeByName() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const headerRowName = sheet.names.getItem("ExpensesHeader");
+                const headerRange = headerRowName.getRange();
+                headerRange.format.fill.color = "red";
```

Comment:
Don't know if you saw my earlier comment about the color "red".  I bet there is a more visually-appealing hex color.  And it's good to show hex being used, too...